tests/test_configDictField.py

```diff
@@ -113,7 +110,7 @@ def testSave(self):
         rt.load("emptyConfigDictTest.py")
         os.remove("emptyConfigDictTest.py")
 
-        self.assertEqual(rt.d1, None)
+        self.assertIsNone(rt.d1, None)
```

Comment:
Kind of confusing to leave the `None` in here (and below), when it's actually a message argument.